bigbluebutton-html5/imports/ui/components/audio/audio-notification/container.jsx

```diff
@@ -1,140 +1,21 @@
 import { createContainer } from 'meteor/react-meteor-data';
+import Service from '/imports/ui/components/audio/service';
 import React, { Component } from 'react';
-import { defineMessages, injectIntl } from 'react-intl';
 import AudioNotification from './component';
-import AudioManager, { AudioErrorCodes } from '/imports/api/2.0/audio/client/manager';
 
-const intlMessages = defineMessages({
-  [AudioErrorCodes.CODE_1001]: {
-    id: 'app.audioNotification.audioFailedError1001',
-    description: 'Audio connection failed with error 1001',
```

Comment:
reading through this pull request im not sure i see the replacement for these error codes. I see lots of good error handling but can you explain where this gets handled? i see at the bottom you return an error code. I don't see where these 11 error messages get used